File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/FrostNucleus.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item;

/* JADX INFO: loaded from: classes3.dex */
public class FrostNucleus extends Item {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.item_frost_nucleus_name;
        this.idDescription = R.string.item_frost_nucleus_description;
        this.idImage = R.drawable.frost_nucleus;
        this.source.add(Integer.valueOf(R.string.dungeon_name_frostbite_peaks));
        this.price = 80L;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/FrostNucleus.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item

class FrostNucleus : Item() {
    override fun configureProperties() {
        idName = R.string.item_frost_nucleus_name
        idDescription = R.string.item_frost_nucleus_description
        idImage = R.drawable.frost_nucleus
        source.add(R.string.dungeon_name_frostbite_peaks)
        price = 80L
    }
}
